src/builder/cleanup.js

```diff
@@ -0,0 +1,10 @@
+import { join } from "path"
+import { unlinkSync } from "fs"
+
+export default (wd, fileToKeep, assets) => {
+  assets
+    .filter((file) => file !== fileToKeep)
+    .forEach((file) => {
+      unlinkSync(join(wd, file))
```

Comment:
I'll leave it there for now. This will be a lot easier if we use async/await for `src/builder/index.js`